src/DynamoCoreWpf/ViewModels/PackageManager/Docs/PublishPackageDocumentation.html

```diff
@@ -269,7 +268,6 @@ <h4>Adding package files</h4>
             When in doubt, toggle the setting off to allow Dynamo to set up the folder structure.
         </p>
     </p>
-    -->
```

Comment:
The description says it's an advanced setting. Is it still an advanced setting now that it's the default?

Reply:
@hwahlstrom @QilongTang should we remove the advanced tag?

Reply:
@zeusongit make sense

Reply:
updated